server/src/rest/snakesRouter.ts

```diff
@@ -6,15 +6,15 @@ import * as _ from 'lodash'
 export const router = Router()
 
 router.get('/', ensureAuthenticated, async (req: express.Request, res: express.Response) => {
-    const snakes = _.range(0, 10).map(i => {
+    const snakes = _.range(0, 11).map(i => {
         return {
-            name: `SWU Training Snake ${i}`,
+            name: `Training Snake ${i}`,
             url: `https://battlesnake-training-snakes.herokuapp.com/snake_${i}`
         }
     })
 
     snakes.push({
-        name: "DSnek",
```

Comment:
Why the rename?

Reply:
to make it seem consistent, as if they are all training snakes. seemed odd to have one differently named.

Reply:
sorry, my mistake. undoing.